run/send-push-message

```diff
@@ -58,11 +58,12 @@ function send_sns () {
 function send_telegram () {
   log "Sending Telegram message for moved files."
```

Comment:
This should either be inside the `if` block, or the `[ -n "${TELEGRAM_ENABLED+x}" ]` test should be moved to line 87, replacing the `[ -r "/root/.teslaCamTelegramSettings" ]` that you removed.

Reply:
I think I missed a commit when squashing.. Should be fixed now.